src/modules/headset/headset_openxr.c

```diff
@@ -381,6 +388,16 @@ static bool openxr_init(float supersample, float offset, uint32_t msaa) {
       .systemId = state.system
     };
 
+    XrSessionCreateInfoOverlayEXTX overlayInfo = {
+      .type = XR_TYPE_SESSION_CREATE_INFO_OVERLAY_EXTX,
+      .next = info.next,
```

Comment:
Thoughts on leaving `sessionLayersPlacement` at 0?  Not sure how that should be handled, or just left as a future improvement.

Reply:
Fields might be in flux, once stabilised we can add support for them.